source/Ubuntu-B.ufo/fontinfo.plist

```diff
@@ -7,7 +7,7 @@
 		<key>capHeight</key>
 		<integer>693</integer>
 		<key>copyright</key>
-		<string>Copyright 2011 Canonical Ltd.  Licensed under the Ubuntu Font Licence 1.0</string>
+		<string>Copyright 2011 Canonical Ltd.</string>
```

Comment:
Probably useful to leave the libre licence for the moment.  It is included in the GF release, so changing this would increase the diff size.